README.md

```diff
@@ -1,6 +1,7 @@
 # Qiskit
+
 [![License](https://img.shields.io/github/license/Qiskit/qiskit.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
-[![Release](https://img.shields.io/github/release/Qiskit/qiskit.svg)](https://github.com/Qiskit/qiskit/releases)
+[![Release](https://img.shields.io/github/release/Qiskit/qiskit.svg?logo=Qiskit)](https://github.com/Qiskit/qiskit/releases) ![GitHub Release](https://img.shields.io/github/v/release/Qiskit/qiskit?sort=semver&filter=0.*&logo=Qiskit&label=extended%20support)
```

Comment:
Very very minor, but potential tweaks:
- could the alt text for "Release" now be "Current Release", and the alt text for the new badge be "Extended Support Release" or something like that?
- can you put the new badge on a separate line of the file (it'll still just appear with a single space in the output), just since that's how all the rest are.
- is there a reasonable hyperlink we ought to make for the new badge?  Doesn't seem like it, but just if you know of one.

Reply:
good points. Addressed in 8b6a1d5312eb07ae5cc22980ae5e2aaf97e8167c